Assets/SEE/Utils/SwitchVersionControlSystems.cs

```diff
@@ -0,0 +1,26 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
```

Comment:
This class must be contained in the namespace `SEE.Utils`.

Reply:
Why is this class contained in `SEE.Utils` anyhow? It don't see why it could be generally re-usable component. Isn't it merely specific to `DiffCity`? Or shouldn it be part of a new layer abstracting from the version control system?

Reply:
Please take a look, if its better now.

Reply:
Fine. The class could be made static. I fixed that.